etc/test-helper.js

```diff
@@ -10,3 +10,4 @@ var deprecationMessage = 'The etc/test-helper.js module has been deprecated. Use
 exports.initSyncFunction = deprecate(testHelper.initSyncFunction, deprecationMessage);
 exports.initDocumentDefinitions = deprecate(testHelper.initDocumentDefinitions, deprecationMessage);
 exports.init = deprecate(testHelper.initSyncFunction, deprecationMessage);
+exports.validationErrorFormatter = require('./validation-error-message-formatter.js');
```

Comment:
As I mentioned in the pull request description, this must be included to prevent errors when a client is using the deprecated test-helper module and they try to use functions that have been deprecated and do not exist in the new validation-error-formatter module (e.g. `supportedExtensionsAttachmentViolation`).